syntax = "proto3";

message Insult {
    string value = 1;
}

message Insults {
    repeated string value = 1;
}

message Empty {}

service InsultingService {
    rpc AddInsult(Insult) returns (Empty) {}
    rpc GetInsults(Empty) returns (Insults) {}
    rpc Insultme(Empty) returns (Insult) {}
}